spec/finalization-group.html

```diff
@@ -148,7 +148,8 @@ <h1>FinalizationGroup.prototype.cleanupSome ( [ _callback_ ] )</h1>
         *TypeError* exception.
         1. If _finalizationGroup_ does not have a [[Cells]]
         internal slot, throw a *TypeError* exception.
-        1. If IsCallable(_callback_) is *false*, throw a *TypeError* exception.
+        1. If _callback_ is not *undefined* and IsCallable(_callback_) is
+        *false*, throw a *TypeError* exception.
```

Comment:
this typically wouldn't be line-broken in the middle; ie there's no need to manually word-wrap.

Reply:
I like to follow that convention myself, but I'm just trying to stick with the conventions @gsathya set for this file. We can adopt other conventions when merging into the main spec.